Sources/windows/main.cc

```diff
@@ -0,0 +1,301 @@
+#include <cmath>
+#include <cstdint>
+#include <napi.h>
+#include <iostream>
+#include <shtypes.h>
+#include <string>
+#include <windows.h>
+// #include <filesystem>
+#include <psapi.h>
+#include <version>
+#include <dwmapi.h>
+#include <algorithm>
+
+typedef int(__stdcall *lp_GetScaleFactorForMonitor)(HMONITOR, DEVICE_SCALE_FACTOR *);
+
+struct OwnerWindowInfo
+{
+  std::string path;
+  std::string name;
+};
+
+template <typename T>
+T getValueFromCallbackData(const Napi::CallbackInfo &info, unsigned handleIndex)
+{
+  return reinterpret_cast<T>(info[handleIndex].As<Napi::Number>().Int64Value());
+}
+
+// Get wstring from string
+std::wstring get_wstring(const std::string str)
+{
+  return std::wstring(str.begin(), str.end());
+}
+
+std::string getFileName(const std::string &s)
+{
+  char sep = '/';
+#ifdef _WIN32
+  sep = '\\';
+#endif
+  size_t i = s.rfind(sep, s.length());
```

Comment:
Don't use abbreviated variable names. Applies everywhere.
```suggestion
  size_t index = string.rfind(separator, string.length());
```